src/js/utils/stylesheet.js

```diff
@@ -0,0 +1,16 @@
+import document from 'global/document';
+
+export let getStyleElement = function(className) {
```

Comment:
Make sense. I guess it could possibly look to see if a style element with the same class name exists and return that and create one, otherwise, or something, but I don't think that's that useful currently.